app.moosync.moosync.yml

```diff
@@ -0,0 +1,54 @@
+app-id: app.moosync.moosync
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "21.08"
+command: moosync
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
```

Comment:
also `home` is a broad filesystem permission.

Reply:
Quite a lot of people have their audio files stored outside of xdg-music. Although if it is unacceptable i could restrict it to xdg-music